hoomd/mpcd/pytest/test_geometry.py

```diff
@@ -56,6 +56,49 @@ def test_pickling(self, simulation_factory, snap):
         pickling_check(geom)
 
 
+class TestRotatedParallelPlates:
+
+    def test_default_init(self, simulation_factory, snap):
+        geom = hoomd.mpcd.geometry.RotatedParallelPlates(separation=8.0,
+                                                         angle=45)
+        assert geom.separation == 8.0
+        assert geom.angle == 45
+        assert geom.speed == 0.0
+        assert geom.no_slip
+
+        sim = simulation_factory(snap)
+        geom._attach(sim)
+        assert geom.separation == 8.0
+        assert geom.angle == 45
+        assert geom.speed == 0.0
+        assert geom.no_slip
+
+    def test_nondefault_init(self, simulation_factory, snap):
+        geom = hoomd.mpcd.geometry.RotatedParallelPlates(separation=10.0,
+                                                         angle=45,
+                                                         speed=1.0,
+                                                         no_slip=False)
+        assert geom.separation == 10.0
+        assert geom.angle == 45
+        assert geom.speed == 1.0
+        assert not geom.no_slip
+
+        sim = simulation_factory(snap)
+        geom._attach(sim)
+        assert geom.separation == 10.0
+        assert geom.angle == 45
+        assert geom.speed == 1.0
+        assert not geom.no_slip
+
+    def test_pickling(self, simulation_factory, snap):
+        geom = hoomd.mpcd.geometry.RotatedParallelPlates(separation=8.0)
```

Comment:
I think this test will fail because it's missing `angle` as an argument, right? Did you try running these locally with `pytest`? See the directions I sent Kwabena for how to do so.